deployment/ansible/roles/model-my-watershed.geoprocessing/templates/upstart-akka-http.conf.j2

```diff
@@ -0,0 +1,13 @@
+description "Akka HTTP Server"
+
+start on runlevel [3]
+stop on shutdown
+
+kill timeout 20
```

Comment:
I believe it was. I think either solution works here (explicit reordering or setting of dependency).